### PR TITLE
Create Login_test

### PR DESCRIPTION
**Pull request by kenkirito**
## Issue that this pull request solves

 Closes: # (issue number)

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by jackfrost13** [comment]
Login cannot be an authorised API. One doesn't need to provide the token. To be more precise, one will never have a valid JWT before login

**Comment by kenkirito** [comment]
ok

File: postman/Login_test [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+									"    pm.response.to.have.status(200);\r",
+									"});\r",
+									"pm.test(\"email\", function () {\r",
+									"    pm.expect(pm.response.text()).to.include(\"lm.mishra56@gmail.com\");\r",
+									"});\r",
+									"\r",
+									"const jsonData = pm.response.json();\r",
+									"pm.test(\"Test data type of the response\", () => {\r",
+									"  pm.expect(jsonData.email).to.be.a(\"string\");\r",
+									"  pm.expect(jsonData.password).to.be.a(\"string\");\r",
```
Comment (jackfrost13) [inline postman/Login_test:28]: password is not returned in the response. One of your tests was failing due to this

File: postman/Login_test [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"auth": {
+							"type": "bearer",
```
Comment (jackfrost13) [inline postman/Login_test:37]: This request must not contain auth header

**Comment by kenkirito** [comment]
so i need to use bear token . i have also checked all end pts and I can't find any end point that contain password in it

File: postman/Login_test [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+								"login"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		}
+	]
+}
```
Comment (jackfrost13) [inline postman/Login_test:75]: You need to save the response token into a local variable named `TOKEN`

File: postman/Login_test [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Valid request\", function () {\r",
+									"    pm.response.to.have.status(200);\r",
+									"});\r",
+									"pm.test(\"email\", function () {\r",
+									"    pm.expect(pm.response.text()).to.include(\"lm.mishra56@gmail.com\");\r",
```
Comment (jackfrost13) [inline postman/Login_test:22]: This should not be hard coded into the test

File: postman/Login_test [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+		"_postman_id": "cdaee4f6-9167-4df8-8e54-3f10e5dcdab5",
+		"name": "Login_Form",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Post",
+			"item": [
+				{
+					"name": "API test",
```
Comment (jackfrost13) [inline postman/Login_test:12]: name should be login

File: postman/Login_test [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+							"port": "3500",
+							"path": [
+								"auth",
+								"login"
+							]
+						}
+					},
+					"response": []
+				}
+			]
```
Comment (jackfrost13) [inline postman/Login_test:72]: You should include more than one request, demonstrating several use cases. Some of them could be

- 400 for invalid request
- Invalid email or password
And supporting test cases for them

**Review by jackfrost13 — CHANGES_REQUESTED** [review]

**Comment by jackfrost13** [comment]
> so i need to use bear token . i have also checked all end pts and I can't find any end point that contain password in it

No, this API will not contain any auth header. One gets the auth token only once they log in.

**Comment by jackfrost13** [comment]
Please go through my comments. Try to incorporate more sample requests demonstrating the several use cases.
let me know if you need more inputs from my end.
Re request for a review whenever you require one

**Comment by kenkirito** [comment]
ok thanks, I will try to complete it by tommorrow

**Pull request closed without merging**